AsyncDisplayKit/ASTextNode.mm

```diff
@@ -237,6 +237,19 @@ - (ASTextKitRenderer *)_renderer
   return [self _rendererWithBounds:self.threadSafeBounds];
 }
 
+- (ASTextKitRenderer *)_rendererWithBoundsSlow:(CGRect)bounds
+{
+  ASDN::MutexLocker l(__instanceLock__);
+
+  CGSize constrainedSize = bounds.size;
+  if (_constrainedSize.width == -INFINITY) {
```

Comment:
See my comment below. This `ASTextNode` addition is just a temporary solution for now and should be pulled out from this PR.